omniauth-tiktok-oauth2.gemspec

```diff
@@ -12,6 +12,6 @@ Gem::Specification.new do |gem|
   gem.files         = `git ls-files`.split("\n")
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'omniauth-oauth2', '~> 1.6'
-  gem.add_runtime_dependency 'oauth2', '~> 1.1'
+  gem.add_runtime_dependency 'omniauth-oauth2', '~> 1.8'
+  gem.add_runtime_dependency 'oauth2', '>= 2.0.7'
```

Comment:
Maybe we should lock this from version 3 as well? I guess omni-oauth2 already has that upper guard though.

Reply:
Yes, we only indirectly depend on `oauth2` it is up to `omni-oauth2` to restrict it properly. No need to restrict it more than necessary.